pubspec.lock

```diff
@@ -667,4 +660,4 @@ packages:
     source: hosted
     version: "2.1.16"
 sdks:
-  dart: ">=2.4.0 <3.0.0"
+  dart: ">=2.3.0 <3.0.0"
```

Comment:
Why the downgrade here?

Reply:
Hmm.. I've been using the 2.4.0 SDK... It seems like pub might be using the largest range possible (consistent with the SDK constraints for all the packages)